aptos-move/framework/aptos-framework/sources/apt_primary_fungible_store.move

```diff
@@ -0,0 +1,77 @@
+
+module aptos_framework::apt_primary_fungible_store {
+    use aptos_framework::fungible_asset::{Self, Metadata, BurnRef};
+    use aptos_framework::primary_fungible_store;
+    use aptos_framework::object;
+
+    use std::signer;
+
+    friend aptos_framework::aptos_account;
+    friend aptos_framework::transaction_fee;
+    friend aptos_framework::transaction_validation;
+
+    inline fun store_address(account: address): address {
```

Comment:
styling: Can you move the entry and public functions to the top, followed by public(friend) and with the private functions at the end? This would make it much easier to navigate

Reply:
sure, didn't know that styling guidance